website/src/pages/guides/continuous-developer-onboarding/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../../../components/Layout'
+import { useHubSpot } from '../../../hooks/hubSpot'
+import styles from './index.module.scss'
+
+const ContinuousDevOnboarding: FunctionComponent<PageProps> = props => {
+    useHubSpot('na1', '2762526', '35e18409-5be7-4fcb-aa57-8152b34eef66', `form`, true)
+
+    return (
+        <Layout
+            location={props.location}
+            meta={{
+                title: 'Guide to Developer Onboarding | Accelerate Dev Onboarding',
+                description:
+                    'Continuous onboarding shifts traditional onboarding practices to cultivate an environment that enables developers to build themselves as flexible and resilient engineers.',
+            }}
+            hero={<div className={styles.hero} />}
+            className="bg-white"
+        >
+            <div className="container py-lg-6 py-4">
+                <div className="row">
+                    <div className="col-lg-6 pr-lg-6">
+                        <h3 className="display-3">
```

Comment:
It seems like we need more of a margin here to match the design spec. Here's what I see currently:
![Screen Shot 2022-03-21 at 10 16 45 AM](https://user-images.githubusercontent.com/43454804/159281734-0b26da53-728b-4155-a1fa-ff37d937ee87.png)

and the design:
![Screen Shot 2022-03-21 at 10 24 25 AM](https://user-images.githubusercontent.com/43454804/159281842-b19ceeb1-0342-4266-b392-e3b41506c12d.png)

Reply:
Oh! Looks like some changes were made to the design since I last looked. I'll push up some updates!